app.js

```diff
@@ -0,0 +1,140 @@
+exit.addEventListener('click', closeMenu);
+
+// Select Players Section
+const players = document.querySelector('.players');
+const body = document.querySelector('.body');
+body.style.backgroundColor = '#ebe5c9';
+
+// Create Content for Players Section
+const playerDetails = [
+  {
+    Picture: './images/stars/luca.png',
+    Name: 'Luca Doncic',
+    Team: 'Dallas Mavericks',
+    Position: 'PG',
+    Bio: 'Nicknamed "El Matador", He will dominate you with his power, athlecism and insane court vision.',
+    class: 'star-1',
+  },
+  {
+    Picture: './images/stars/giannis.png',
+    Name: 'Giannis Antetokounpo',
+    Team: 'Milwaukee Bucks',
+    Position: 'SF',
+    Bio: 'Called the Greek Freak, he is a force to be reckoned with in the NBA.',
+    class: 'star-2',
+  },
+  {
+    Picture: './images/stars/curry.jpeg',
+    Name: 'Steph Curry',
+    Team: 'Golden State Warriors',
+    Position: 'PG',
+    Bio: 'This Guy will hurt you from anywhere and you cannot do anything about it. He is the all-time leader in three points made.',
+    class: 'star-3',
+  },
+  {
+    Picture: './images/stars/lebron.png',
+    Name: 'LeBron James',
+    Team: 'LA Lakers',
+    Position: 'SF',
+    Bio: 'Regarded by some as the Greatest of all Time, LeBron is not one you mess with. He is King James.',
+    class: 'star-4',
+  },
+  {
+    Picture: './images/stars/jokic.png',
+    Name: 'Nikola Jokic',
+    Team: 'Denver Nuggets',
+    Position: 'C',
+    Bio: 'Called the Joker, do not let his tricks fool you. When he means business, he means business.',
+    class: 'star-5',
+  },
+  {
+    Picture: './images/stars/tatum.png',
+    Name: 'Jayson Tatum',
+    Team: 'Boston Celtics',
+    Position: 'SF',
+    Bio: 'This Guy is amazing at what he does. He can drop points however he likes.',
+    class: 'star-6',
+  }];
+
+// Create Function to Populate the Section
+function fillPlayer() { /* eslint-disable-next-line */
+  for (let i = 0; i < playerDetails.length; i++) {
+    const container = document.createElement('div');
+    container.className = 'container';
+    container.setAttribute('id', playerDetails[i].class);
+    const picture = document.createElement('img');
+    picture.className = 'star-img';
+    picture.setAttribute('src', playerDetails[i].Picture);
+    const starBio = document.createElement('div');
+    starBio.className = 'star-bio';
+    const name = document.createElement('h3');
+    name.className = 'star-name';
+    name.innerHTML = playerDetails[i].Name;
+    const team = document.createElement('p');
+    team.className = 'team';
+    team.innerHTML = playerDetails[i].Team;
+    const position = document.createElement('p');
+    position.className = 'position';
+    position.innerHTML = playerDetails[i].Position;
+    const bio = document.createElement('p');
+    bio.className = 'bio';
+    bio.innerHTML = playerDetails[i].Bio;
+
+    // Append name, position and bio to starBio
+    starBio.appendChild(name);
```

Comment:
- [ ] This section is supposed to look like the one we have on the design give please try to add all the necessary details for it to like the design given PLease look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/183119372-cba2e976-ff2a-4f74-bfc1-350d443b2569.png)

**design given**

![image](https://user-images.githubusercontent.com/90222110/183119615-61691034-e1a3-49fa-a71f-fc669b06b203.png)

Reply:
- [ ] Just as the previous reviewer mentioned, and to try to follow the design guidelines, please make sure to work on this section and try to match the design as much as possible. Don't forget about the little box with some black and white squares, that is also part of the design!